packages/cli/src/util/extension/proxy.ts

```diff
@@ -8,10 +8,7 @@ import {
 import type { Server } from 'http';
 import type Client from '../client';
 
-const toHeaders = buildToHeaders({
-  // @ts-expect-error - `node-fetch` Headers is missing `getAll()`
-  Headers,
-});
+const toHeaders = buildToHeaders({ Headers });
```

Comment:
I don't understand how this TS error was fixed <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" width="20" height="auto" align="absmiddle"></picture>

Reply:
Upgrading `@edge-runtime/node-utils` fixed the issue I think because of this PR: https://github.com/vercel/edge-runtime/pull/309